ktor-io/jvm/test/io/ktor/utils/io/BlockingAdaptersOnProhibitedThreadTest.kt

```diff
@@ -42,12 +40,10 @@ class BlockingAdaptersOnProhibitedThreadTest {
     }
 
     @Test
-    fun testOutputStreamProhibited() {
+    fun testOutputStreamProhibitedNotFails() {
```

Comment:
It's better to name a test without negation like `canConvertChannelToOutputStreamInNetworkThread`